packages/slack-app/Readme.md

```diff
@@ -5,56 +5,11 @@ This app i built using [node slack sdk](https://slack.dev/node-slack-sdk/). Read
 
-Follow these steps to open this project in a development container:
+### Dependencies
 
-1. If this is your first time using a development container, please follow the [getting started steps](https://aka.ms/vscode-remote/containers/getting-started).
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
-2. In Visual Studio Code, press <kbd>F1</kbd> and select the **Remote-Containers: Open Folder in Container...** command. Select the cloned copy of this folder, wait for the container to start, and try things out!
-
-## Start Development environment using docker-compose
-
-The first time this command is ran it will download and start the local developmment environment.
-
-The command also starts a container running [ngrok](https://ngrok.com), which is used to expose the Alvtime-slack-app on a public url. This url is useful to connect the slack app configured on at [https://api.slack.com/apps](https://api.slack.com/apps) and test it in a development slack workspace. The public url is shown in the ngrok inspector.
-
-Start the environment:
-
-```
-docker-compose -f .devcontainer/docker-compose.yaml up -d
-```
-
-Start the verification server
-
-```
-docker-compose -f .devcontainer/docker-compose.yaml exec alvtime-slack-app npx slack-verify --secret $(grep SLACK_SIGNING_SECRET .env | cut -d '=' -f2)
-```
-
-- The ngrok inspector is available on `localhost:4040`
-
-Copy the url in the ngrok inspector and use it as the new request URL under Event Subscriptions on the api.slack.com app setup page to trigger the verification. Stop the verification server and run the following to start the app in watch mode
-
-```
-docker-compose -f .devcontainer/docker-compose.yaml exec alvtime-slack-app npm run watch
-```
-
-To stop and remove the containers run
-
-```
-docker-compose -f .devcontainer/docker-compose.yaml down --volumes
-```
-
-To view the backend logs run.
-
-```
-docker-compose -f .devcontainer/docker-compose.yaml logs -f
-```
-
-Download the latest backend api and database images
-
-```
-docker-compose -f .devcontainer/docker-compose.yaml pull
-```
+The only way we know how to develop the slack-app is by using the docker-compose setup. The dev environment needs some secrets we have chosen to store in a azure key vault. To get access to this key vault login using `az login`. You can now run `./run slack-app` from the root of the alvtime project. This command will start the necessary services and drop you into a bash shell in the slack app development service. Her you can run `npm run watch` to start the development server. Navigate to the [development slack workspace](dev-qje3117.slack.com/) and log in useng Ahre-Ketil-lillehagen-on-dev-qje3117-slack-com credentials found in our common test user keyvault. In the slack workspace you can run `/alvtime tasks` in anny channal. This will prompt a login flow. Use Ahre-Ketil's azure account to login to azure. The credentials can be found in the same keyvault.
```

Comment:
Et par skriveleifer: "Her you can run" -> "Here you can run", "anny channal" -> "any channel"

Reply:
Done :)